src/arch/arm/rte_atomic.h

```diff
@@ -0,0 +1,10 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2015 RehiveTech. All rights reserved.
+ */
+
+#ifndef _RTE_ATOMIC_ARM_H_
+#define _RTE_ATOMIC_ARM_H_
+
+#include "arch/arm/rte_atomic_64.h"
```

Comment:
same here what is the need of rte_atomic_64.h, we can have one file. Ideally we include rte_atomic.h in the code and here in this header file we decide which file to include. Have #ifdef to decide which one to include based on the architecture (32 bit or 64 bit) if there is a difference in the implementation for both architecture.

Reply:
yes, good point